web/src/main/java/org/springframework/security/web/session/ContinueRequestSessionInformationExpiredStrategy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.session;
+
+import jakarta.servlet.ServletException;
+import java.io.IOException;
+
+/**
+ * A {@link SessionInformationExpiredStrategy} continues to execute subsequent filters
+ * even after session expiration.
+ *
+ * @author Rosie Yang
```

Comment:
Please add `@since 6.3` too

Reply:
Thanks! I changed too. I realized I need to pay more attention to contribution rules, sorry about that.